### PR TITLE
Pavel-Latyshou/To update landing page for each `Crypto E-Book` based on language

### PR DESCRIPTION
**Pull request by pavel-latyshou-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/landing/ebooks/crypto.js [hunk 1]
```diff
@@ -27,10 +28,32 @@ const query = graphql`
...
+            ...bannerImage
+        }
     }
 `
 
-const StocksEbook = () => {
+const StocksEbook = (props) => {
+    const { language } = props
+
+    console.log(language)
```
Comment (habib-deriv) [inline src/pages/landing/ebooks/crypto.js:55]: Let's remove this console

File: src/pages/landing/ebooks/crypto.js [hunk 2]
```diff
@@ -52,7 +75,9 @@ const StocksEbook = () => {
                 ebook_utm_code="cryptocurrencies-ebook"
                 introSub=""
                 introMain={localize('Learn how to trade cryptocurrencies with Deriv')}
-                mainHeaderImage={data['crypto_hero']}
+                mainHeaderImage={
+                    language === 'en' ? data[`crypto_hero`] : data[`crypto_hero_${language}`]
```
Comment (habib-deriv) [inline src/pages/landing/ebooks/crypto.js:79]: Not all language have different photos. What will happen for italiano or polish ?

File: src/pages/landing/ebooks/crypto.js [hunk 3]
```diff
@@ -66,7 +91,9 @@ const StocksEbook = () => {
             />
             <Topics
                 title={localize('In this e-book we will cover')}
-                topicsImage={data['crypto_inside']}
+                topicsImage={
+                    language === 'en' ? data[`crypto_inside`] : data[`crypto_inside_${language}`]
```
Comment (habib-deriv) [inline src/pages/landing/ebooks/crypto.js:95]: Same comment as above

**Review by habib-deriv — APPROVED** [review]

**Comment by sean-binary** [comment]
Please help with the conflict @pavel-latyshou-deriv and please run all these images through this [site](https://compresspng.com). They are nearly 2mb each.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**